express/properties/workflow.py

```diff
@@ -1,27 +1,234 @@
+                "cls": "User",
+                "slug": ""
+            },
+            "owner": {
+                "_id": "",
+                "cls": "Account",
+                "slug": ""
+            },
+            "schemaVersion": "0.2.0",
+            "exabyteId": "",
+            "hash": "",
+            "_id": "",
+        }
+        return config
+
+    def _serialize(self) -> dict:
+        config = self.get_common_config()
+        config.update(self.get_workflow_specific_config())
+        return config
+
+
+class PyMLTrainAndPredictWorkflow(WorkflowProperty):
+    """
+    Quick implementation of the new version of ExabyteML. We expect workflows to have a format as follows:
+
+    Workflow_Head_Subworkflow - Contains various units which prepare an ML job. For example, we may have the following
+    units present.
+        - An Assignment unit specifying whether the workflow is in Train or Predict mode (head-set-predict-status)
+        - A Conditional unit that specifies whether the train or predict setup is to be used
+        - Training setup: An assignment unit to specify the training data to be included
+        - Training setup: An IO unit to copy the training data into the current working directory
+        - Predict setup: An assignment unit to specify the data to perform a prediction on
+        - Predict setup: An IO unit to copy in the predict data to the current working directory
+        - Predict setup: An IO unit to copy in any files necessary for the predict workflow to function
+                        (head-fetch-trained-model)
+
+    The final IO unit in the predict setup that we discuss, which copies in files needed for the workflow to function,
+    is populated by this class's _create_download_from_object_storage_input function. It obtains a list of files
+    the user has deemed important-enough to save in the workflow, and ensures they're around for the predict job.
+
+    Workflow_Tail_Subworkflow - Contains the actual ML that the user wants to perform. The intention is that this is
+    the more user-modifiable part of the ML feature. We might contain the following units here:
+        - pyml:setup_variables_packages: A setup file that helps the user communicate the files needed in subsequent
+                                         predict runs
+        - pyml:data_input:read_csv:pandas: Uses Pandas to read in a CSV for use in further ML
+        - pyml:pre_processing:standardization:sklearn: Scales the data such that it has mean=0 and variance=1, then
+                                                       saves the scaler for use in predict workflows
+        - pyml:model:multilayer_perceptron:sklearn: A multilayer perceptron being fit to a regression problem. Saves
+                                                    the model to be used again in predict workflows
+        - pyml:post_processing:parity_plot:matplotlib: Creates a parity plot if the workflow is in "Training" mode.
+    """
```

Comment:
I think it will be helpful to add the actual JSON file for the workflow - as extracted from webapp. Maybe not in the docstring, and maybe not in git but in LFS, and maybe as a test asset, but it can be helpful in the future

Reply:
Added the samples to /tests/fixtures/pyML/test-001/